src/components/testCaseLanding/common/TestCase.tsx

```diff
@@ -256,7 +256,7 @@ const TestCase = ({
                   export collection bundle
                 </button>
               </>
-            ) : (
+            ) : featureFlags.exportTestCase ? (
```

Comment:
Rather than a check on a feature flag, we'd like to do a null check on the export function prop that comes in. For QI-Core it will be set, and for QDM, it will not be set. This way we can avoid yet another feature flag to manage!

Reply:
Referring to the `exportTestCase` prop